File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        // Is nums sorted? 
        int n = nums.size(); 
        unordered_map<int, int> freqMap; 
        for(int i = 0; i < n; i++)
            freqMap[nums[i]]++; 
        for(int num: nums){
            if (freqMap[num] > 1)
                return true; 
        } 
        return false; 
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq_map = {} 
        for num in nums: 
            freq_map[num] = 1 + freq_map.get(num, 0)
        eleMap = []
        for key, v in freq_map.items(): 
            eleMap.append([v, key])
        eleMap.sort()
        res = []
        while(len(res) < k): 
            res.append(eleMap.pop()[1])
        return res
        